src/flutter/templates.ts

```diff
@@ -25,7 +25,10 @@ export function initSnippet(
 ): string {
   let snippet = `await SentryFlutter.init(
     (options) {
-      options.dsn = '${dsn}';`;
+      options.dsn = '${dsn}';
+      // Adds request headers and IP for users, for more info visit:
+      // https://docs.sentry.io/platforms/dart/guides/flutter/data-management/data-collected/ for more info
```

Comment:
`l` A bit redundant having fore more info after already stating more info before the link. Applies to all changes in this file.
```suggestion
      // Adds request headers and IP for users, for more info visit:
      // https://docs.sentry.io/platforms/dart/guides/flutter/data-management/data-collected/
```